app/cdap/components/shared/ConfigurationGroup/index.tsx

```diff
@@ -138,6 +143,11 @@ const ConfigurationGroupView: React.FC<IConfigurationGroupProps> = ({
       values: newValues,
     };
     setFilteredConfigurationGroups(newFilteredConfigurationGroup);
+    // to trigger another update if properties are updated with default values
+    // only trigger if the contents of two objs are different
+    if (!isEqual(newValues, updatedValuesWithDefault)) {
+      changeParentHandler(updatedValuesWithDefault);
```

Comment:
Looks like we missed some code? 
<img width="624" alt="image" src="https://user-images.githubusercontent.com/20428112/165163180-a4e4f279-e860-4706-89f9-ec53687571df.png">